staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -49,5 +50,5 @@ func SetupCurrentKubernetesSpecificFeatureGates(featuregates featuregate.Mutable
 // To add a new feature, define a key for it at k8s.io/api/pkg/features and add it here.
 var cloudPublicFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	IPv6DualStack:                    {Default: true, PreRelease: featuregate.Beta},
-	ControllerManagerLeaderMigration: {Default: false, PreRelease: featuregate.Alpha},
+	ControllerManagerLeaderMigration: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I believe this should also exist in pkg/features/kube_features.go. (ala IPv6DualStack)

Reply:
We have this discussion when working on alpha. The conclusion was that `ControllerManagerLeaderMigration` is a local feature flag to `staging/controller-manger` thus does not need a place in `pkg/features/kube_features.go`. I do not see a reason to inverse such conclusion during alpha-to-beta graduation. Could you illiterate a bit more why we need it in `pkg/features/kube_features.go` now?

Reply:
I don't believe we did have a discussion on this in alpha. You did in fact make this assertion during alpha (https://github.com/kubernetes/kubernetes/pull/99417#issuecomment-789958998) however I did not agree then and still do not. However as an alpha feature I was unwilling to block based on that alone. For beta I do consider this to be blocking. pkg/feature/kube_features is the master list and if its missing from there it will be missing from places like https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/.

Reply:
Thanks for pointing out the doc. Updating now.